libpod/container_log.go

```diff
@@ -81,5 +83,45 @@ func (c *Container) readFromLogFile(options *logs.LogOptions, logChannel chan *l
 		}
 		options.WaitGroup.Done()
 	}()
+	// Check if container is still running or paused
+	if options.Follow {
+		eventChannel := make(chan *events.Event)
+		go func() {
+			readOpts := events.ReadOptions{
+				EventChannel: eventChannel,
+				Filters:      []string{"type=container"},
+				FromStart:    false,
+				Stream:       true,
+			}
+			eventsError := c.runtime.Events(readOpts)
+			if eventsError != nil {
+				logrus.Error(eventsError)
+				close(eventChannel)
```

Comment:
@jwhonce Does this conflict with your changes to events for APIv2?

Reply:
I did a rebase with the changes just merged for events for APIv2. Running through checks now.